modules/effects/src/main/scala/dev/profunktor/redis4cats/interpreter/Redis.scala

```diff
@@ -897,6 +900,31 @@ private[redis4cats] class BaseRedis[F[_]: ContextShift, K, V](
     JRFuture {
       async.flatMap(c => F.delay(c.keys(key)))
     }.map(_.asScala.toList)
+
+  override def info: F[Map[String, String]] =
+    JRFuture {
+      async.flatMap(c => F.delay(c.info))
+    }.map(
```

Comment:
I feel like this could blow up if something goes wrong with Redis response and looks like a side-effect. Let's protect from it wrapping it in F.

```scala
.flatMap { data =>
  F.delay {
    data.split ... 
  }
}